ui/src/components/Header/Header.js

```diff
@@ -0,0 +1,28 @@
+import { user } from './Mock';
+
+export default function Header() {
+  return (
+    <header
+      className='py-2 px-8 h-20 flex items-center bg-rds-primary text-slate-100'
+    >
+      <img
+        src={user.picture.url}
+        className='rounded-full h-12 block mx-2'
+        alt='User Picture'
+      />
+      <div
```

Comment:
Use text tag here such as `Heading` or `Paragraph`. Ideally, this should be a `Heading` tag. Instead of `DIV`